api/src/auth/drivers/ldap.ts

```diff
@@ -111,7 +111,7 @@ export class LDAPAuthDriver extends AuthDriver {
 				{
 					attributes: ['cn'],
```

Comment:
Can you try to add `distinguishedName` over here? It should contain the DN.

Reply:
I got it in a nicer way 😄